docs/pages/getting-data-with-pelican/client.mdx

```diff
@@ -241,6 +241,91 @@ The Pelican client supports a variety of command line flags that modify the clie
 - **-r or --recursive:** Takes no argument and indicates to Pelican that all sub paths at the level of the provided namespace should be copied recursively. This option is only supported if the origin supports the WebDav protocol.
 - **-t or --token:** Takes a path to a file containing a signed JWT, and is used to download protected objects.
 
+## Listing Objects on an Origin
+
+To use the pelican client to list objects in a collection, use Pelican's `object ls` sub-command
+
+```bash
+pelican object ls pelican://<federation-url></namespace-prefix></path/to/collection>
+```
+
+This will only be successful when listings are enabled in the Pelican origin. To learn more about origin capabilities, go to the [Origin docs](../federating-your-data/origin.mdx).
+
+If listings are not enabled, you will see the following error:
+
+```
+ERROR[2025-02-07T15:27:05-06:00] Failure getting pelican://osdf-director.osg-htc.org:443/nrao-ardg: failed to perform list request: Collections URL not found in director response. Are you sure there's an origin for prefix /<namespace-prefix> that supports listings?
+```
+
+## Getting Object Data From a Federation with Stat
+
+To use the pelican client to get object data, use Pelican's `object stat` sub-command.
+
+```bash
+pelican object stat pelican://<federation-url></namespace-prefix></path/to/object>
+```
+
+This will return data in the form of:
+
+```
+Name: </namespace-path></path/to/object>
+Size: <size>
+ModTime: <time-last-modified>
+IsCollection: false
+```
+
+This command can also be run on a collection:
+
+```bash
+pelican object stat pelican://<federation-url></namespace-prefix></path/to/collection>
+```
+
+and the data will look like:
+
+```
+Name: </namespace-path></path/to/collection>
+Size: 0
+ModTime: <time-last-modified>
+IsCollection: true
+```
+
+## Syncing Objects with an Origin
+
+Pelican also has an `object sync` command which will sync objects from a source object or collections to a destination object or collection. If the objects already exist in the destination and are the same size or larger than the source objects of the same name, they will not be overwritten, but the objects which don't exist will be added.
+
+> **Warning**: In the case where an object in the destination is smaller than an object with the same name in the source, the sync command assumes that a partial sync was interrupted and completes it by appending the necessary bytes. If it issn't the same file, but has the same name, this can result in the mashing of two files together.
+
+### Sync Objects from a Collection to a Local Directory
```

Comment:
Few comments:
- Can you provide some discussion of or a link to how this can be done with tokens?
-  I think `sync` from remote to local also requires that `listings` is enabled, doesn't it? Can you explain any assumptions/requirements on Origin/prefix capabilities and how the user can verify those?
- I can't remember all the implementation details, but are there any other assumptions to document about any importance of trailing `/`s in anything? I know you had a big chart to explain all the operations `sync` had to perform. Maybe that table is even worth including here?